File: spring-boot-integration/postgres-demo-app/src/main/java/io/github/mfvanek/pg/spring/postgres/PostgresDemoApplication.java
/*
 * Copyright (c) 2021-2024. Ivan Vakhrushev.
 * https://github.com/mfvanek/pg-index-health-test-starter
 *
 * This file is a part of "pg-index-health-test-starter".
 *
 * Licensed under the Apache License 2.0
 */

package io.github.mfvanek.pg.spring.postgres;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;

@SpringBootApplication
public class PostgresDemoApplication {

    /**
     * Demo application with PostgreSQL datasource.
     */
    public static void main(final String[] args) {
        SpringApplication.run(PostgresDemoApplication.class, args);
    }
}

File: spring-boot-integration/kotlin-demo-app/src/main/kotlin/io/github/mfvanek/pg/spring/postgres/kt/PostgresDemoApplication.kt
/*
 * Copyright (c) 2021-2024. Ivan Vakhrushev.
 * https://github.com/mfvanek/pg-index-health-test-starter
 *
 * This file is a part of "pg-index-health-test-starter".
 *
 * Licensed under the Apache License 2.0
 */

package io.github.mfvanek.pg.spring.postgres.kt

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication

@SpringBootApplication
class PostgresDemoApplication

fun main(args: Array<String>) {
    runApplication<PostgresDemoApplication>(*args)
}
